server/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "server",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
```

Comment:
can you please make the start script with nodemon which will hot reload the app? Add nodemon in dev dependencies.